### PR TITLE
Subscribe support

### PR DESCRIPTION
**Pull request by Perondas**
I implemented the subscribe support like [in this PR](https://github.com/versatica/JsSIP/pull/711) on the JsSIP repo.
Differences are that I did not implement the Notifier class.

### CODE REVIEW

**Comment by Perondas** [comment]
This also closes #203

**Comment by ghenry** [comment]
Awesome!!!!!!

File: lib/src/sip_ua_helper.dart [hunk 1]
```diff
@@ -2,8 +2,12 @@ import 'dart:async';
 
 import 'package:flutter_webrtc/flutter_webrtc.dart';
 import 'package:logger/logger.dart';
+import 'package:sip_ua/src/event_manager/internal_events.dart';
```
Comment (cloudwebrtc) [inline lib/src/sip_ua_helper.dart:5]: Using relative paths here will make the code look neater.

File: lib/src/subscriber.dart [hunk 2]
```diff
@@ -0,0 +1,564 @@
+import 'dart:async';
+import 'dart:math';
+
+import 'package:collection/collection.dart';
+import 'package:sip_ua/sip_ua.dart';
```
Comment (cloudwebrtc) [inline lib/src/subscriber.dart:5]: Same here, I prefer relative paths, it looks like less code, :)

Comment (Perondas) [reply lib/src/subscriber.dart]: Is this better?

Comment (cloudwebrtc) [reply lib/src/subscriber.dart]: This makes it easier to distinguish between internal imports or external package imports, I think.

Comment (Perondas) [reply lib/src/subscriber.dart]: Makes sense, I am farily new to dart, and as such I dont know all the finder details.

**Comment by cloudwebrtc** [comment]
thanks, it looks good

**Comment by datpt11** [comment]
how can i use it

**Pull request merged**